caluma/workflow/models.py

```diff
@@ -39,7 +43,12 @@ class Workflow(SlugModel):
     meta = JSONField(default=dict)
     is_published = models.BooleanField(default=False)
     is_archived = models.BooleanField(default=False)
-    start = models.ForeignKey(Task, on_delete=models.CASCADE, related_name="+")
+    start = models.ForeignKey(
+        Task,
+        on_delete=models.CASCADE,
+        related_name="+",
+        help_text="Task workflow will be started with.",
```

Comment:
```suggestion
        help_text="First task of the workflow.",
```

Reply:
How about the text change?

Reply:
ouch with my force pushed I have lost this change... :smile: Let me readd it again.

Reply:
done